### PR TITLE
20489 Misc cleanup

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#20489

*Description of changes:*
- app version = 5.9.11
- saved correct name option for other filings (not used but denotes code intent for future debugging/audit)
- updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-674-dprzede5.web.app

**Review by JazzarKarim — APPROVED** [review]
There's a mismatch between ticket name and PR 😆

LGTM Sev! 👍

File: src/mixins/filing-template-mixin.ts
Comment (severinbeauvais) [inline src/mixins/filing-template-mixin.ts:1]: Although the subject ticket is not for work in Create UI, I'm trying to make Edit UI similar to Create UI, and this change is to clean up / make consistent some Create UI code.

Specifically, the changes here are to be intentional in the "correct name option" that we are saving in a draft filing, even though these particular filings don't restore this value. Basically, I don't like that we have to infer what option the user has selected by looking at which properties exist or not.

Amalgamations, continuation ins (WIP) and restorations do restore the correct name option value. I think this is a better design.

**Comment by severinbeauvais** [comment]
> There's a mismatch between ticket name and PR 😆

Thanks for checking that. Yes, here I go back to Create UI to clean up something I noticed while comparing with other filings and with Edit UI.

**Comment by severinbeauvais** [comment]
PS There's still an issue uploading coverage to CodeCov but the tests pass fine locally and even here.

Locally:
image

The "Test with Vitest" step passed:
image

**Comment by JazzarKarim** [comment]
> PS There's still an issue uploading coverage to CodeCov but the tests pass fine locally and even here.
> 
Yup, been having this issue as well in my PRs. All good.

**Review by ketaki-deodhar — APPROVED** [review]
Consistent code across UIs will make it easier for us to work. Thank you for this. Looks good Sev!

**Pull request merged**